connector_oxigesti/components/deleter.py

```diff
@@ -20,3 +20,40 @@ class OxigestiExportDeleter(AbstractComponent):
     def run(self, external_id):
         adapter = self.component(usage="backend.adapter")
         adapter.delete(external_id)
+
+
+class OxigestiBatchExportDeleter(AbstractComponent):
+    _name = "oxigesti.batch.export.deleter"
+    _inherit = ["base.exporter", "base.oxigesti.connector"]
+
+    def run(self, filters=None):
```

Comment:
```suggestion
    def run(self, domain=None):
```
Do this (replace `filters` by `domain`) only in the new code everywhere